src/content/en/updates/2018/03/clipboardapi.md

```diff
@@ -0,0 +1,269 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Copycats rejoice! Async Clipboard API is unblocking copy & paste in Chrome 66.
+
+{# wf_updated_on: 2018-03-13 #}
+{# wf_published_on: 2018-03-13 #}
+{# wf_tags: cutandcopy,execcommand,input,chrome66 #}
+{# wf_featured_image: /web/updates/images/generic/share.png #}
+{# wf_featured_snippet: Async Clipboard API simplifies permissions-friendly copy & paste. #}
+{# wf_blink_components: Blink>Input #}
+
+
+# Unblocking Clipboard Access {: .page-title }
+
+{% include "web/_shared/contributors/developit.html" %}
+
+
+Over the past few years, browsers have converged on using
+[document.execCommand] for clipboard interactions. It's great to have a single
+widely-supported way to integrate copy and paste into web apps, but this came at
+a cost: clipboard access is synchronous, and can only read & write to the DOM.
+
+Synchronous copy & paste might seem fine for small bits of text, but there are a
+number of cases where blocking the page for clipboard transfer leads to a poor
+experience. Time consuming sanitization or decoding might be needed before
+content can be safely pasted.  The browser may need to load linked resources
+from a pasted document - that would block the page while waiting on the disk or
+network. Imagine adding permissions into the mix, requiring that the browser
+block the page while asking the user if an app can access the clipboard.
+
+At the same time, the permissions put in place around `document.execCommand` for
+clipboard interaction are loosely defined and vary between browsers. So, what
+might a dedicated clipboard API look like if we wanted to address blocking and
+permissions problems?
+
+That's the new [Async Clipboard API], the text-focused portion of which we're
+shipping in Chrome 66.  It's a replacement for execCommand-based copy & paste
+that has a well-defined permissions model and doesn't block the page.  This new
+API also Promises (see what I did there?) to simplify clipboard events and align
+them with the [Drag & Drop API].
+
+<div style="display: flex; justify-content: center">
```

Comment:
why wrap this in flex?

Reply:
I copied the video embed from Surma's CSS Paint update, not sure why he did it.